Added closing quotation mark

JeroenOortwijn opened the pull request:
Not as bad as https://xkcd.com/859/, but still disturbing. ;)

JeroenOortwijn committed:
Added closing quotation mark

Not as bad as https://xkcd.com/859/, but still disturbing. ;)

mypil commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

cmcatee-MSFT reviewed (changes requested):
Please move the closing quotation mark outside the ending period. It should read:

...send changes from "Contoso Support" or "Building A Reception Desk."

JohanFreelancer9 commented on microsoft-365/admin/email/about-shared-mailboxes.md:
```diff
@@ -26,7 +26,7 @@ description: "Shared mailboxes are used when multiple people need access to the
 Shared mailboxes are used when multiple people need access to the same mailbox, such as a company information or support email address, reception desk, or other function that might be shared by multiple people.
 
-Users with permissions to the group mailbox can send as or send on behalf of the mailbox email address if the administrator has given that user permissions to do that. This is particularly useful for help and support mailboxes because users can send emails from "Contoso Support" or "Building A Reception Desk.
+Users with permissions to the group mailbox can send as or send on behalf of the mailbox email address if the administrator has given that user permissions to do that. This is particularly useful for help and support mailboxes because users can send emails from "Contoso Support" or "Building A Reception Desk".
```
```suggestion
Users with permissions to the group mailbox can send as or send on behalf of the mailbox email address if the administrator has given that user permissions to do that. This is particularly useful for help and support mailboxes because users can send emails from "Contoso Support" or "Building A Reception Desk."
```

JeroenOortwijn committed:
Update microsoft-365/admin/email/about-shared-mailboxes.md

Apparently in American English, periods and commas always go inside quotation marks.

Further reading: https://www.quickanddirtytips.com/education/grammar/how-to-use-quotation-marks

Every day you learn something new! :D

Co-Authored-By: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>
